example/lib/main.dart

```diff
@@ -51,9 +51,8 @@ void main() async {
         TitleBarStyle.hidden,
         windowButtonVisibility: false,
       );
-      await windowManager.setSize(const Size(755, 545));
```

Comment:
Maybe we don't need to make use of `.setSize`. We can set the window size from the windows runner. That is a better solution, that works at runtime and hot restart isn't going to break it